data_override/include/data_override_impl.inc

```diff
@@ -351,8 +274,14 @@ endif
  if(use_get_grid_version .EQ. 2) then
    call close_file(fileobj)
  end if
+end subroutine DATA_OVERRIDE_INIT_IMPL_
+
+function count_ne_1(in_1, in_2, in_3)
```

Comment:
This function needs documentation

Reply:
Doxygen comment added in 59bf15d438dca05b9481e4372e8616b7794c3476.